azure-pipelines.yml

```diff
@@ -15,25 +6,31 @@ pr:
 - master
 
 # Three phases for each of the three OSes we want to run on
-phases:
-- template: /eng/common/templates/phases/base.yml
+jobs:
+- template: /eng/common/templates/job/job.yml
   parameters:
     name: Windows_NT
+    enableMicrobuild: true
+    enablePublishBuildArtifacts: true
+    enablePublishTestResults: true
```

Comment:
you probably don't have test results (maybe you do).  if not, delete this

Reply:
Impressively, we do, 27 of 'em. :-) How exciting.